pandas/core/methods/selectn.py

```diff
@@ -90,39 +92,42 @@ def is_valid_dtype_n_method(dtype: DtypeObj) -> bool:
 
 class SelectNSeries(SelectN[Series]):
     """
-    Implement n largest/smallest for Series
+    Implement n-sorting for Series
 
     Parameters
     ----------
     obj : Series
     n : int
-    keep : {'first', 'last'}, default 'first'
+    keep : {'first', 'last', 'all'}, default 'first'
 
     Returns
     -------
     nordered : Series
     """
 
-    def compute(self, method: str) -> Series:
+    def compute(self, ascending: bool | Sequence[bool]) -> Series:
         from pandas.core.reshape.concat import concat
 
+        if isinstance(ascending, Sequence):
+            assert len(ascending) == 1
+            ascending = ascending[0]
+
         n = self.n
         dtype = self.obj.dtype
         if not self.is_valid_dtype_n_method(dtype):
-            raise TypeError(f"Cannot use method '{method}' with dtype {dtype}")
+            raise TypeError(f"Cannot use n-sorting with dtype {dtype}")
```

Comment:
I think this is okay but wonder if `Cannot sort dtype {dtype}` might be more clear?

Reply:
I tend to agree with your point. I updated the message.